float32[] cpg_parameters
---
string ParamMessageResponse
---
string ParamMessageFeedback
